tests/BaseArrayCollectionTest.php

```diff
@@ -331,7 +332,7 @@ public function testMatchingWithSortingPreserveKeys(): void
                 'object1' => $object1,
             ],
             $collection
-                ->matching(new Criteria(null, ['sortField' => Criteria::ASC]))
+                ->matching(new Criteria(null, ['sortField' => Order::ASC]))
```

Comment:
By simply changing the tests, you lose coverage for the old way of using this API. Codecov rightfully complains about that.